migrate/v0_16/testdata/appstate-bank-v15.json

```diff
@@ -0,0 +1,541 @@
+          "address": "kava1g33w0mh4mjllhaj3y4dcwkwquxgwrma9ga5t94",
+          "coins": [
+            {
+              "amount": "1000000000000",
+              "denom": "ukava"
+            }
+          ],
+          "public_key": null,
+          "sequence": "0"
+        }
+      },
+      {
+        "type": "cosmos-sdk/Account",
+        "value": {
+          "account_number": "0",
+          "address": "kava1ynf22ap74j6znl503a56y23x5stfr0aw5kntp8",
+          "coins": [
+            {
+              "amount": "1000000000000",
+              "denom": "ukava"
+            }
+          ],
+          "public_key": null,
+          "sequence": "0"
+        }
+      },
+      {
+        "type": "cosmos-sdk/Account",
+        "value": {
+          "account_number": "0",
+          "address": "kava1acge4tcvhf3q6fh53fgwaa7vsq40wvx6wn50em",
+          "coins": [
+            {
+              "amount": "1000000000000",
+              "denom": "ukava"
+            }
+          ],
+          "public_key": null,
+          "sequence": "0"
+        }
+      },
+      {
+        "type": "cosmos-sdk/Account",
+        "value": {
+          "account_number": "0",
+          "address": "kava173w2zz287s36ewnnkf4mjansnthnnsz7rtrxqc",
+          "coins": [
+            {
+              "amount": "1000000000000",
+              "denom": "ukava"
+            }
+          ],
+          "public_key": null,
+          "sequence": "0"
+        }
+      },
+      {
+        "type": "cosmos-sdk/Account",
+        "value": {
+          "account_number": "0",
+          "address": "kava1fwfwmt6vupf3m9uvpdsuuc4dga8p5dtl4npcqz",
+          "coins": [
+            {
+              "amount": "1000000000000",
+              "denom": "ukava"
+            }
+          ],
+          "public_key": null,
+          "sequence": "0"
+        }
+      },
+      {
+        "type": "cosmos-sdk/Account",
+        "value": {
+          "account_number": "0",
+          "address": "kava1sw54s6gq76acm35ls6m5c0kr93dstgrh6eftld",
+          "coins": [
+            {
+              "amount": "1000000000000",
+              "denom": "ukava"
+            }
+          ],
+          "public_key": null,
+          "sequence": "0"
+        }
+      },
+      {
+        "type": "cosmos-sdk/Account",
+        "value": {
+          "account_number": "0",
+          "address": "kava1t4dvu32e309pzhmdn3aqcjlj79h9876plynrfm",
+          "coins": [
+            {
+              "amount": "1000000000000",
+              "denom": "ukava"
+            }
+          ],
+          "public_key": null,
+          "sequence": "0"
+        }
+      },
+      {
+        "type": "cosmos-sdk/Account",
+        "value": {
+          "account_number": "0",
+          "address": "kava1wuzhkn2f8nqe2aprnwt3jkjvvr9m7dlkpumtz2",
+          "coins": [
+            {
+              "amount": "1000000000000",
+              "denom": "ukava"
+            }
+          ],
+          "public_key": null,
+          "sequence": "0"
```

Comment:
nit: all the `sequence` values are 0 which can hide bugs. If the migrations got the tag name wrong they'd look for a nonexistent field, and fill in the value with the default 0. I'd maybe change a couple to something other than 0.